cyw43-firmware/src/lib.rs

```diff
@@ -0,0 +1,349 @@
+#![no_std]
+
+pub const BLUETOOTH_FIRMWARE_OFFSETS: [(u32, usize); 28] = [
```

Comment:
I had to sort of kind of manually extract these from a `pico-sdk` example with lots of log patches so it'd make maintaining future Bluetooth firmware changes a pain...